libsolidity/analysis/TypeChecker.cpp

```diff
@@ -475,7 +475,7 @@ void TypeChecker::checkContractExternalTypeClashes(ContractDefinition const& _co
 	for (auto const& it: externalDeclarations)
 		for (size_t i = 0; i < it.second.size(); ++i)
 			for (size_t j = i + 1; j < it.second.size(); ++j)
-				if (!it.second[i].second->hasEqualParameterTypes(*it.second[j].second))
+				if (!it.second[j].second->canOverwrite(*it.second[i].second))
```

Comment:
Since we call `interfaceFunctionType` to get this, `hasEqualParameterTypes` should be sufficient here.

Reply:
Ah I see, we actually don't use `interfaceFunctionType`, we just check if it exists.